scripts/post.lua

```diff
@@ -0,0 +1,22 @@
+local pipelineDepth = 15
```

Comment:
It's 15 in the sample pipeline.lua from wrk. 1 in the pipeline.lua in the same dir as this in the repo.